src/60_AMP_Ads/Banner_Ad.html

```diff
@@ -8,43 +8,33 @@
 
   This sample demonstrates how to implement a simple banner [AMP Ad](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md). In general an AMP Ad should be a valid AMP document.
 
-  The code represents the body that should be returned as a response initiated through an [amp-ad](/components/amp-ad/) component or otherwise whenever we want to return an AMP Ad in any other context.
-
-  This example consists of an image and a rendering tracking pixel that are implemented respectively using `amp-img` and `amp-pixel` components.
--->
 <!-- -->
 <!doctype html>
+<!-- This tells everyone that this is an AMP for Ads file. -->
 <html ⚡4ads>
 <head>
-
+  <!-- The charset definition must be the first child of the `<head>` tag. -->
   <meta charset="utf-8">
+  <!-- The AMP runtime should be loaded as the second child of the `<head>` tag.-->
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <!-- ## Setup -->
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <!--
-  As part of any other AMP document, we import the canonical boiler plate to make it a valid AMP document.
+    The AMP ad boilerplate.
   -->
-  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <style amp4ads-boilerplate>body{visibility:hidden}</style>
+  <!--
+    CSS must be embedded inline.
+  -->
+  <style amp-custom>
+    h1 {
+      color: red;
+    }
+  </style>
 </head>
 <body>
-
-  <!-- ## Basic Usage -->
   <!--
-  The anatonomy of banner ad is an image with a hyperlink to the advertiser's page. We display the image for the creative using [amp-img](/components/amp-img/).
+    AMP Ads use the same AMPHTML components as normal AMP pages. 
```

Comment:
AMP ads are only allowed to use a restricted set of components as described in the spec.